ietf/api/tests_ietf_utils.py

```diff
@@ -0,0 +1,46 @@
+# Copyright The IETF Trust 2025, All Rights Reserved
+
+from django.test import TestCase, RequestFactory
```

Comment:
We have a custom `TestCase` class - import it as `from ietf.utils.test_utils` instead of `django.test`

Reply:
Fixed with d89138baba5ad29a87718e66e2bf787dbe9f449a